ulib/FStar.Seq.Equiv.fst

```diff
@@ -0,0 +1,75 @@
+module FStar.Seq.Equiv
```

Comment:
Would be good for this to have an interface, which only shows the signature of eq_of_seq, the statements of the lemmas, and the main seq_equiv definition.

Reply:
All done